static/js/popover_menus.js

```diff
@@ -92,6 +92,17 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        ...default_popover_props,
+        target: "#create-stream-link",
```

Comment:
@jai2201 I think we should point the user directly to the "Create a stream" popover when the user has no streams they can subscribe, as pointed out in the issue: `same place as the "Create a stream" link from the "+" menu next to the STREAMS header.`

Reply:
Ahh - I think I misunderstood the comment there!
Thanks for pointing this out!

Reply:
Fixed, Thanks !